flottform/where-are-you-at/src/routes/feedback/+server.ts

```diff
@@ -0,0 +1,57 @@
+import { CREATE_ISSUE_TOKEN, CREATE_ISSUE_URL } from '$env/static/private';
+import { json } from '@sveltejs/kit';
+
+const REPO_URL = `${CREATE_ISSUE_URL}/issues`;
+const accessToken = CREATE_ISSUE_TOKEN || '';
+
+const sanitizeUserInput = (text: string): string => {
+	return text
+		.replace(/([\\_*[\](){}\-#`])/g, '\\$1')
+		.replace(/</g, '&lt;')
+		.replace(/>/g, '&gt;');
+};
+
+export const POST = async ({ fetch, request }) => {
+	const {
+		userName,
+		email,
+		linkedin,
+		twitter,
+		feedbackPositive,
+		feedbackImprovements,
+		contactChoice
+	} = await request.json();
+	const issueBody = `**Name**: ${sanitizeUserInput(userName)}
+**📧 E-Mail**: ${sanitizeUserInput(email) || '-'}
+**📟 LinkedIn**: ${sanitizeUserInput(linkedin) || '-'}
+**🐦 X / Twitter**: ${sanitizeUserInput(twitter) || '-'}
+**Preferred way of contact**: ${sanitizeUserInput(contactChoice)}
+
+**🚀 What they like about Flottform:**
```

Comment:
We should change 'Flottform' to 'where-are-you.at' or 'Location Tracker'

```suggestion
**🚀 What they like about where-are-you.at:**
```

Reply:
I kept Flottform because I thought we built `where-are-you.at` to show the power and capabilities of Flottform and thus the feedback would be for Flottform.

But that's also ok, I can change it.

Reply:
You are right.. I'm just a bit worried that it may be confusing for user. If someone wants to give feedback and share ideas about both products (like Return and complains form and where-are-you.at), open both forms in two tabs and then can't really tell what form is that, unless he looks at url 😅